src/components/scenes/SendScene2.tsx

```diff
@@ -826,13 +828,15 @@ const SendComponent = (props: Props) => {
           walletId
         })
       }
-      Alert.alert(lstrings.transaction_success, lstrings.transaction_success_message, [
-        {
-          onPress() {},
-          style: 'default',
-          text: lstrings.string_ok
-        }
-      ])
+      if (dismissAlert !== true) {
```

Comment:
Now that `dismissAlert` is a true boolean, we can do `if (!dismissAlert)`, which is much easier to understand.